src/command-line/install.js

```diff
@@ -21,6 +21,8 @@ program
 			return;
 		}
 
+		log.info("Checking requested package data on npm...");
```

Comment:
What about `Retrieving information about the package, please wait...`?

Reply:
Perhaps `Retrieving information about the package...` ?

Reply:
👍